Maths/QuadraticFormula.js

```diff
@@ -4,28 +4,26 @@
  * @param {number} a
  * @param {number} b
  * @param {number} c
- * @return {string}
+ * @return {array}
  * @see https://en.wikipedia.org/wiki/Quadratic_formula
- * @example quadraticFormula(1, -3, -4) = 'the roots are X1=4 and X2=-1'
- * @example quadraticFormula(1, 5, 6) = 'the roots are X1=-2 and X2=-3'
- * @example quadraticFormula(1, -3, 8) = 'the roots do not exist or the roots are imaginary'
+ * @example quadraticFormula(1, -3, -4) = [4, -1]
+ * @example quadraticFormula(1, 5, 6) = [-2, -3]
+ * @example quadraticFormula(1, -3, 8) = []
  */
 
-const quadraticFormula = (a, b, c) => {
+const solveQuadraticEquation = (a, b, c) => {
   if (typeof a !== 'number' || typeof b !== 'number' || typeof c !== 'number') {
     return new TypeError('Some argument is not a number.')
   }
   const discriminant = (b ** 2) - (4 * a * c)
   const denominator = 2 * a
-  let answer = ''
+  let answer = []
   if (discriminant >= 0) {
```

Comment:
Please distinguish `discriminant == 0` here to only add a single item to `answer` in that case.